test/fixtures/basic/unmanaged-pod.yml.erb

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: unmanaged-pod-<%= deployment_id %>
+  labels:
+    type: unmanaged-pod
+    name: unmanaged-pod-<%= deployment_id %>
+    app: basic
+spec:
+  activeDeadlineSeconds: 20
+  restartPolicy: Never
+  containers:
+    - name: basic
+      image: hello-world:latest
```

Comment:
This is a docker tutorial thing: [docker hub](https://hub.docker.com/_/hello-world/) / [repo](https://github.com/docker-library/hello-world). It just prints a message and exits, which is exactly what we need this test pod to do. Wdyt? Other ideas?